packages/kampus-ui/src/ThemeProvider.tsx

```diff
@@ -1,27 +1,25 @@
 import type { FC } from "react";
 import { createContext, useContext, useMemo, useState } from "react";
 
-type ThemeVariants = "light" | "dark";
+type ThemeVariants = "LIGHT" | "DARK";
```

Comment:
[later] find a way to make this generic so we can use the generated enum types from prisma in pano.